storyblok/storyblok-design-system
# refactor: sb pagination to template syntax

## Pull request type

Jira Link: -




## How to test this PR

## Pull request type

Jira Link: -




## How to test this PR


Compare:

- https://blok.ink/?path=/docs/design-system-components-sbpagination--default
- 


It's also good to check in the app:
```cd storyblok-design-system && git checkout refactor/ar-28-sb-pagination```
Then start the server of the app



## What is the new behavior?



- pagination uses the template syntax
- 
- 

## Other information
